spec/requests/schools/partnerships_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Schools::Partnerships", type: :request do
```

Comment:
I'm a bit short of time to add the cucumber, cypress tests in this pr given todays deadline but I'll be following up with another pr where they will be addressed.

Reply:
Have you made a ticket for that?

Reply:
I'll do them now that the demo is gone